packages/tasit-act/src/tasitAct.test.js

```diff
@@ -0,0 +1,83 @@
+import Contract from "./tasitAct";
+import { expect, assert } from "chai";
+// Note: This file is originally genarated by `tasit-contracts` and was pasted here manually
```

Comment:
```suggestion

// Note: SimpleStorage.json is originally genarated by `tasit-contracts` and was pasted here manually
```

Reply:
Saying "The SimpleStorage ABI" might be clearer than "this file", because we might mean the file this text is in